### PR TITLE
Translate "Getting Started in Chrome's Origin Trials" into Japanese

### PR DESCRIPTION
**Pull request by yoichiro**
I have translated the [Getting started with Chrome's origin trials - Chrome Developers](https://developer.chrome.com/blog/origin-trials/) article into Japanese.

Post: [Getting started with Chrome's origin trials - Chrome Developers](https://developer.chrome.com/blog/origin-trials/)
Translator: @yoichiro 
Reviewer: @agektmr

### CODE REVIEW

**Comment by yoichiro** [comment]
@agektmr Could you review this pull request? Thank you in advance!

File: site/ja/blog/origin-trials/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+-  [Process for launching new features in Chromium](https://www.chromium.org/blink/launching-features)
+-  [Intent to explain: Demystifying the Blink shipping process](https://www.youtube.com/watch?time_continue=291&v=y3EZx_b-7tk)
+-  [What are third-party origin trials?](/blog/third-party-origin-trials/)
+-  [View origin trials information in DevTools](/blog/new-in-devtools-94/#origin-trials)
+---
+
+Photo by [Louis Reed
+](https://unsplash.com/@_louisreed) on [Unsplash](https://unsplash.com/photos/pwcKF7L4-no).
+
+Translated by [yoichiro](https://github.com/yoichiro).
```
Comment (chybie) [inline site/ja/blog/origin-trials/index.md:139]: We can follow the What's New in DevTools article format, which we mentioned *翻訳者の yoichiro さん ... さんに感謝いたします。* at the beginning of the article

File: site/ja/blog/origin-trials/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+authors:
+  - samdutton
+date: 2020-06-22
+updated: 2021-09-01
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/KeaVCdXHWzrI35QRvsZL.jpg
+alt: Pipette with purple liquid
+tags:
+  - origin-trials
+---
+
```
Comment (chybie) [inline site/ja/blog/origin-trials/index.md:14]: Put the translator here @yoichiro

Comment (yoichiro) [reply site/ja/blog/origin-trials/index.md:14]: @chybie Thank you for the suggestion! I will put the expression based on the What's new in DevTools article on the top of this article.

Comment (yoichiro) [reply site/ja/blog/origin-trials/index.md:14]: @chybie I put it at 104959b.

**Comment by yoichiro** [comment]
@agektmr I have added my name at the top of this article as a translator at 104959b.

File: site/ja/blog/origin-trials/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+registered [origin](https://web.dev/same-site-same-origin/#origin). Third-party origin trials make
+it possible for providers of embedded content to try a new feature across multiple sites
+without requiring a token for every origin. -->
+オリジントライアルは通常、ファーストパーティベースでのみ利用でき、登録された単一の [オリジン](https://web.dev/same-site-same-origin/#origin) でのみ機能します。サードパーティオリジントライアルにより、埋め込みコンテンツのプロバイダーは、オリジンごとにトークンを必要とせずに、複数のサイトを横断して新しい機能を試すことが可能となります。
+
+詳細はこちらをご覧ください: [What are third-party origin trials?](/blog/third-party-origin-trials/)
+
+
+<!-- ## Deprecation trials -->
+## 非推奨トライアル
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:57]:
```suggestion
## デプリケーショントライアル
```
I'm intentionally using a term "デプリケーション トライアル" for this word in our dev blog. Can you please align with it?

File: site/ja/blog/origin-trials/index.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+tags:
+  - origin-trials
+---
+
+*翻訳者の [yoichiro](https://github.com/yoichiro) さんに感謝いたします。*
+
+<!-- Origin trials give you access to a new or experimental feature, to build
+functionality your users can try out for a limited time before the feature
+is made available to everyone. -->
+オリジントライアルにより、新規の、または実験的な機能へのアクセスと、機能が誰でも利用可能になる前に、限定された期間ユーザが試すことのできる機能の構築ができるようになります。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:20]:
```suggestion
オリジントライアルを利用すると、一般公開されるより先に、新しい機能や実験的な機能を期間を限定してユーザに試してもらうことができるようになります。
```

File: site/ja/blog/origin-trials/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+<!-- When Chrome offers an origin trial for a feature, you can register for the trial to enable
+the feature for all users on your [origin](https://web.dev/same-site-same-origin/#origin),
+without requiring them to toggle any flags or switch to an alternative build
+of Chrome (though they may need to upgrade). Origin trials enable developers
+to build demos and prototypes using new features. The trials also help Chrome engineers
+understand how new features are used, and how they may interact with other web technologies. -->
+Chrome がある機能のオリジントライアルを提供している時は、あなたの [オリジン](https://web.dev/same-site-same-origin/#origin) 上で全てのユーザにその機能を有効にすするためにトライアルに登録することができます。そのために、Chrome のフラグを切り替えたり代替ビルドに切り替える必要はありません（ただし、アップグレードが必要な場合があります）。オリジントライアルにより、開発者は新しい機能を使用してデモやプロトタイプを作成できます。このトライアルは、Chrome エンジニアが新機能がどのように利用され、そして他のウェブテクノロジとどのように相互に利用されるかを理解するのにも役立ちます。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:28]:
```suggestion
Chrome がある機能のオリジントライアルを提供している時、トライアルに登録して、あなたの [オリジン](https://web.dev/same-site-same-origin/#origin) 上で全てのユーザにその機能を有効にすることができます。そのために、Chrome のフラグを切り替えたり、別のビルドに切り替える必要はありません（ただし、アップグレードが必要な場合があります）。オリジントライアルにより、開発者は新しい機能を使用して、デモやプロトタイプを作成できます。このトライアルは、Chrome のエンジニアが新機能がどのように利用され、そして他のウェブテクノロジとどのように相互に利用されるかを理解するのにも役立ちます。
```

File: site/ja/blog/origin-trials/index.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+<!-- Origin trials are public and open to all developers. They are limited in duration and
+usage. Participation is a self-managed process with limited documentation and support.
+Participants should be willing and able to work relatively independently using the
+documentation available, which, at this stage, will likely be limited to API
+specifications and explainers, though web.dev tries to provide guidance whenever
+possible. -->
+オリジントライアルは、全ての開発者に対して、パブリックであり、オープンです。それらは、期間と使用方法に制限があります。参加は、限られたドキュメントとサポートが準備された自己管理プロセスです。参加者は、利用可能なドキュメントを使用して、比較的独立して作業する意思と能力を備えている必要があります。この段階では、API 仕様とその説明者に限定される可能性がありますが、web.dev は可能な限りガイダンスを提供しようとします。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:36]:
```suggestion
オリジントライアルは、全ての開発者に対して、パブリックであり、オープンです。それらは、期間と使用方法に制限があります。参加は、限られたドキュメントとサポートが準備された自己管理プロセスです。参加者は、利用可能なドキュメントを使用して、比較的独立して作業する意思と能力を備えている必要があります。この段階では、API 仕様とその Explainer に限定される可能性がありますが、web.dev が可能な限りのガイダンスを提供します。
```
"Explainer" is a document prepared before writing a spec - example: https://github.com/samuelgoto/email-otp

File: site/ja/blog/origin-trials/index.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+<!-- If you register for a trial, the Chrome team will periodically ask you for specific
+feedback on your use of the trial feature. Some features may undergo multiple origin
+trials, as learnings are incorporated and adjustments are made. -->
+トライアルに登録すると、Chrome チームは定期的にトライアルの機能の利用に関する具体的なフィードバックを求めます。 一部の機能は、学習が組み込まれ、調整が行われるため、複数のオリジントライアルを受ける可能性があります。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:41]:
```suggestion
トライアルに登録すると、Chrome チームは定期的にトライアル機能の利用に関する具体的なフィードバックを求めます。 機能によっては、フィードバックを取り込み、調整を行うため、オリジントライアルが複数回行われる可能性があります。
```

File: site/ja/blog/origin-trials/index.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+詳細はこちらをご覧ください: [What are third-party origin trials?](/blog/third-party-origin-trials/)
+
+
+<!-- ## Deprecation trials -->
+## 非推奨トライアル
+
+<!-- Not all origin trials are for testing new APIs. Some trials enable a deprecated feature to be
+temporarily re-enabled. These are known as **deprecation trials**, and in some contexts have been
+referred to as "reverse" origin trials. -->
+すべてのオリジントライアルが新しい API のテスト向けであるとは限りません。一部のトライアルでは、廃止された機能を一時的に再度有効にすることができます。これらは **非推奨（Deprecation）トライアル** として知られており、一部のコンテキストでは「逆（reverse）」オリジントライアルと呼ばれています。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:62]:
```suggestion
すべてのオリジントライアルが新しい API のテスト向けであるとは限りません。一部のトライアルでは、廃止された機能を一時的に再度有効にすることができます。これらは **デプリケーショントライアル** として知られており、一部のコンテキストでは「リバース」オリジントライアルと呼ばれています。
```

File: site/ja/blog/origin-trials/index.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+1. フィードバックを提出します。オリジントライアルのサイトを通じてこれを行ってください。このフィードバックは公開されず、Chrome チームの限定されたグループの人々へのみ利用可能となります。各トライアルは、自発的なコミュニティフィードバックへのリンクも提供します。これは通常、GitHub またはその他のパブリックチャネルの機能を指します。
+1. トークンが失効するときは、更新のためのリンクが書かれたメールを受け取ります。更新を行うために、再びフィードバックを提出するよう求められます。
+
+{% Aside 'warning' %}
+<!-- Usually if an API lands unchanged after a successful origin trial, there is a short period between the
+end of the origin trial and the date the implementation ships in the browser when the API will not
+be available. This is by design. If Chrome were to avoid the mandatory total-breakage period, that would
+bias toward also avoiding breakages in the API surface, which are often needed to improve the API.
+The final shipping API might be worse for it. -->
+通常、オリジントライアルが成功した後、API が変更されずに着地した場合、オリジントライアルが終了してから、ブラウザにおいて実装がリリースされる日付まで、API は利用できなくなります。これは意図的です。Chrome が完全な破損期間を必ず回避した場合、API を改善するためにしばしば必要となる API サーフェスの破損も回避する方向にバイアスがかかります。最終的にリリースされる API は、それにより悪くなるかもしれません。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:108]:
```suggestion
通常、オリジントライアルが成功し、API が変更されることなく提供される場合、オリジントライアルが終了してから、ブラウザで実装がリリースされる日付まで、API は利用できなくなります。これは意図的です。Chrome がその機能を利用できない期間を設けなかった場合、API を改善するためにしばしば必要となる API が利用できない可能性も回避する方向にバイアスがかかります。最終的にリリースされる API が、それにより悪くなる可能性もあります。
```

**Review by agektmr — CHANGES_REQUESTED** [review]
Reviewd.

**Comment by yoichiro** [comment]
@agektmr Thank you for the reviewing! I check them.

File: site/ja/blog/origin-trials/index.md (hunk 6 above)
Comment (yoichiro) [reply site/ja/blog/origin-trials/index.md:36]: @agektmr I didn't know the "Explainer". Thank you!

**Comment by yoichiro** [comment]
@agektmr I have just committed all suggestions from you.

File: site/ja/blog/origin-trials/index.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Chromeのオリジントライアル入門
+subhead: オリジントライアルは、新規の、または実験的なウェブプラットフォームの機能をテストするための方法です。また、その機能が全てのユーザ向けに利用可能になる前に、その機能の使いやすさ、実用性、そして有効性についてウェブ標準コミュニティにフィードバックする方法でもあります。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:4]:
```suggestion
subhead: オリジントライアルは、ウェブプラットフォームの新しい機能や実験的な機能が一般公開されるより先に、使いやすさ、実用性、そして有効性についてテストし、ウェブ標準コミュニティにフィードバックできる方法です。
```

File: site/ja/blog/origin-trials/index.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+understand how new features are used, and how they may interact with other web technologies. -->
+Chrome がある機能のオリジントライアルを提供している時、トライアルに登録して、あなたの [オリジン](https://web.dev/same-site-same-origin/#origin) 上で全てのユーザにその機能を有効にすることができます。そのために、Chrome のフラグを切り替えたり、別のビルドに切り替える必要はありません（ただし、アップグレードが必要な場合があります）。オリジントライアルにより、開発者は新しい機能を使用して、デモやプロトタイプを作成できます。このトライアルは、Chrome のエンジニアが新機能がどのように利用され、そして他のウェブテクノロジとどのように相互に利用されるかを理解するのにも役立ちます。
+
+<!-- Origin trials are public and open to all developers. They are limited in duration and
+usage. Participation is a self-managed process with limited documentation and support.
+Participants should be willing and able to work relatively independently using the
+documentation available, which, at this stage, will likely be limited to API
+specifications and explainers, though web.dev tries to provide guidance whenever
+possible. -->
+オリジントライアルは、全ての開発者に対して、パブリックであり、オープンです。それらは、期間と使用方法に制限があります。参加は、限られたドキュメントとサポートが準備された自己管理プロセスです。参加者は、利用可能なドキュメントを使用して、比較的独立して作業する意思と能力を備えている必要があります。この段階では、API 仕様とその Explainer に限定される可能性がありますが、web.dev が可能な限りのガイダンスを提供します。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:36]:
```suggestion
オリジントライアルは、全ての開発者に対してパブリックでありオープンですが、期間と使用方法に制限があります。参加は、限られたドキュメントとサポートのみ用意された自己管理プロセスです。参加者は、利用可能なドキュメントを使用して、比較的独立して作業する意思と能力を備えている必要があります。この段階では、API 仕様とその Explainer に限定される可能性がありますが、web.dev が可能な限りのガイダンスを提供します。
```

File: site/ja/blog/origin-trials/index.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+1. Try out the new feature.
+1. Submit feedback. Do this through the origin trial site. This feedback is
+   not public and is available only to a limited group of people on the Chrome
+   team. Each trial also provides a link for spontaneous community feedback.
+   This typically points to the feature on GitHub or some other public
+   channel.
+1. When your token expires, you will get an email with a renewal link.
+   To do so, you are again asked to submit feedback.
+-->
+1. [list of active trials](https://developers.chrome.com/origintrials/#/trials/active) から、オリジントライアルを選択します。
```
Comment (agektmr) [inline site/ja/blog/origin-trials/index.md:91]:
```suggestion
1. [アクティブなトライアルのリスト](https://developers.chrome.com/origintrials/#/trials/active) から、オリジントライアルを選択します。
```

**Review by agektmr — APPROVED** [review]
LGTM % comments

**Comment by yoichiro** [comment]
@agektmr Thank you for the reviewing and suggestions!

**Comment by agektmr** [comment]
@chybie this is ready to merge.

**Comment by agektmr** [comment]
Actually, I'll just merge this.

**Comment by samdutton** [comment]
@jpchase fyi

**Pull request merged**